src/index.html

```diff
@@ -1,13 +1,285 @@
+            <img class="place__photo place__photo--6" src="./images/photos/6.jpg" alt="Place 5">
+          </div>
+
+          <p class="place__address">
+            <a href="#place1">Marina Palms</a>
+            / North Miami Beach, FL 33140
+          </p>
+        </div>
+
+        <div class="gallery__button-container">
+          <a
+            class="gallery__button button button--secondary"
+            href="#view-all"
+          >
+            View all
+          </a>
+        </div>
+      </div>
+    </section>
+
+    <section class="page__section testimonials" id="testimonials">
+      <div class="container">
+        <h2 class="section-title">Testimonials</h2>
+
+        <div class="testimonials__content">
+          <div class="testimonials__item-1 testimonial">
+            <img
+              src="./images/people/1.jpg"
+              alt="Name1 Lastname1"
+              class="testimonial__photo"
+            >
+
+            <p class="testimonial__name">Leslie Mckinney</p>
+            <p class="testimonial__title">CEO</p>
+
+            <p class="testimonial__text">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+
+          <div class="testimonials__item-2 testimonial">
+            <img
+              src="./images/people/2.jpg"
+              alt="Name2 Lastname2"
+              class="testimonial__photo"
+            >
+
+            <p class="testimonial__name">Leslie Mckinney</p>
+            <p class="testimonial__title">CEO</p>
+
+            <p class="testimonial__text">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+
+          <div class="testimonials__item-3 testimonial">
+            <img
+              src="./images/people/3.jpg"
+              alt="Name3 Lastname3"
+              class="testimonial__photo"
+            >
+
+            <p class="testimonial__name">Leslie Mckinney</p>
+            <p class="testimonial__title">CEO</p>
+
+            <p class="testimonial__text">
+              Nice work, Certificates National Association of Realtors (USA)
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="page__section contact-us" id="contact-us">
+      <div class="container">
+        <h2 class="section-title">Contact us</h2>
+
+        <div class="contact-us">
+          <div class="grid">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <h3 class="contact-us__info-title">Phone</h3>
+                +1 234 5555-55-55
+              </div>
+              <div class="contact-us__info">
+                <h3 class="contact-us__info-title">Email</h3>
+                hello@miami.com
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--4-5">
```

Comment:
instead of `item 4-5` can we have just class `item-1` that takes only one column?
in this case you can render
`item-4 item-1 item-4`
instead of 
`item-1-4 item-5-6 item-7-10`

in grid there is smth like `grid-column: span 1`
![image](https://user-images.githubusercontent.com/29413892/121326061-dbdda200-c91a-11eb-92ff-4f605c4b78ec.png)

Reply:
4-5 is 2 columns
ok can replace 1-1 with just 1